en/docs/concepts/oidc/hybrid-client-profile.md

```diff
@@ -0,0 +1,242 @@
+
+# OpenID Connect Hybrid Flow
+
+WSO2 Identity Server supports the OpenID Connect hybrid flow for
+authentication.
+
+Specifying any of the following ` response_type ` values
+in an authorization request selects the hybrid flow for authentication:
+
+- [code token](#code-token)
+- [code id\_token](#code-id_token)
+- [code id\_token token](#code-id_token-token)
+
+To configure WSO2 Identity Server to support the OpenID Connect hybrid
+flow for authentication, you need to add following to the
+` <IS_HOME>/repository/conf/deployment.toml ` file.
+
+```toml
+[[oauth.custom_response_type]]
+name="code token"
+class="org.wso2.carbon.identity.oauth2.authz.handlers.HybridResponseTypeHandle"
+
+[[oauth.custom_response_type]]
+name="code id_token"
+class="org.wso2.carbon.identity.oauth2.authz.handlers.HybridResponseTypeHandler"
+
+[[oauth.custom_response_type]]
+name="code id_token token"
+class="org.wso2.carbon.identity.oauth2.authz.handlers.HybridResponseTypeHandler"
+```
+
+To understand how the ` response_type ` value specified
+in an authorization request selects the hybrid flow to be the
+authentication flow, let's take a look at the following
+` response_type ` values in detail .
+
+### code token
+
+This ` response_type ` requests a code and an access
+token from the authorization endpoint. 
+ 
+Following is a sample authorization request that uses
+` code token ` as the ` response_type ` :
+
+``` java
+https://localhost:9443/oauth2/authorize?response_type=code token&client_id=<Client ID>&nonce=asd&redirect_uri=http://localhost:8080/playground2/oauth2client&scope=openid
+```
+
+Following is the sample response that you will receive on successful
+authorization:
+
+``` java
+http://localhost:8080/playground2/oauth2client#access_token=1940a308-d492-3660-a9f8-46723cc582e9&code=99b34587-5483-374d-8b25-50485498e761&token_type=Bearer&expires_in=299999&session_state=baae9a71cdabe38b4643b9d59bd9f65ffaf5a9b8c453f4256c085e5a1c57e624.-EA3ZqPzLvsk25CKmt56YA
+```
+
+You can send the code to the token endpoint to request for an access
+token, refresh token and id\_token.
```

Comment:
is the \ in "id_token" given here to escape the underscore character? If so, I saw some places where it is not escaped...should it be added there too?